extension/src/shared/ui-components/FeedBackModal.tsx

```diff
@@ -0,0 +1,156 @@
+import { FormControl } from "@mui/base/FormControl";
+import { Input, inputClasses } from "@mui/base/Input";
+import { TextareaAutosize } from "@mui/base/TextareaAutosize";
+import CancelIcon from "@mui/icons-material/Cancel";
+import {
+  Box,
+  Button,
+  Checkbox,
+  FormControlLabel,
+  IconButton,
+  Link,
+  Modal,
+  Typography,
+  styled,
+} from "@mui/material";
+import { useAtom } from "jotai";
+import { useState } from "react";
+
+import { HelperText } from "../../prototypes/ui-components/HelperText";
+import { Label } from "../../prototypes/ui-components/InputLabel";
+import { showFeedbackModalAtom } from "../../prototypes/view/states/app";
+
+const style = {
+  position: "absolute" as const,
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  width: 420,
+  bgcolor: "background.paper",
+  color: "#000",
+  margin: "auto",
+  padding: "18px 22px",
+};
+
+const FeedBackModal: React.FC = () => {
+  const [showFeedbackModal, setShowFeedbackModal] = useAtom(showFeedbackModalAtom);
```

Comment:
I would like to keep ui-components be reusable UI. So could you do a refactor to use this state in `/shared/view/ui-container/FeedBack` like blow?

```tsx
const FeedBack = () => {
  const [showFeedbackModal, setShowFeedbackModal] = useAtom(showFeedbackModalAtom);
  return <FeedBackModal show={showFeedbackModal} />
}
```